index.js

```diff
@@ -3,6 +3,8 @@ var RSS = require('rss');
 var express = require('express');
 var app = express();
 
+var regex_exclude = process.env.DOCKER_HUB_RSS_REGEX_EXCLUDE || false;
```

Comment:
I wasn't aware of that branch; feel free to close this MR in this case. Thanks! :D